hazelcast-client/src/main/java/com/hazelcast/client/ClientOutOfMemoryHandler.java

```diff
@@ -57,19 +58,16 @@ private static void closeSockets(HazelcastClientInstanceImpl client) {
                 try {
                     connectionManager.shutdown();
                 } catch (Throwable ignored) {
-                    EmptyStatement.ignore(ignored);
+                    ignore(ignored);
                 }
             }
         }
 
         private static void tryShutdown(HazelcastClientInstanceImpl client) {
-            if (client == null) {
```

Comment:
This method call is guarded by `instance instanceof HazelcastClientInstanceImpl` in `onOutOfMemory()`, which doesn't match on `null`